### DISCUSSION TITLE
Probably bricked my camera, super confused what camera version I have

### DISCUSSION THREAD

**Discussion by kwint** [roleoroleo/yi-hack-Allwinner#367] [Q&A] [answered]
Hi, 

I was experiencing this issue: https://github.com/roleoroleo/yi-hack-Allwinner/issues/355

So I tried to manually update the camera a few weeks back. Somewhere along the process I had to leave and then the power to the camera is cut automatically (ha automation, I use the camera to check my 3d prints, is off when its not printing)

It was at a state where it would connect to my wifi (I could ping the device) but I couldnt access the devices via SSH or port 808. So I tried to start the update from scratch, but right now its not even connecting to the wifi anymore. the yellow / orange led starts blinking almost directly after boot.

Now to be fully honest at this point I'm fully certain anymore which camera version I have. But since I had the same issue as the person above and there is a y25ga sticker on the device, I was thinking I had that version.

As a last resort I tried putting a few versions on but with no result. So pretty save to assume that my camera is bricked.

So I opened the camera casing to see if I can see the chip inside to determine which version I had. To my surprise the PCB says this device is a y20ga. 

Any tips or tricks I could try before giving this device a permanent place in my bin?

And yes, I should have taken a backup, I know.

![image](https://user-images.githubusercontent.com/7907280/187790554-93636077-4bff-4a0e-be5d-f678ee340564.png)

![image](https://user-images.githubusercontent.com/7907280/187790577-5567aba1-763b-4562-92f5-89fdb1ac2f26.png)

**Comment by roleoroleo** [comment, marked as answer]
The label in the pcb is not enough to know the model but probably a y20ga or y25ga as you already said.
If you can ping the cam probably you can login with telnet (not ssh).
Try it and let me know.

**Reply by kwint** [reply]
Thanks for you response,

I can't seem to the get the camera connected to my network anymore. When I flash another version of the firmware the camera seems to be doing something (blue light turns on for a while, then reboots), then the yellow led starts blinking again.

What does that mean?

This is what the SD card looks like after:
```
├── backup
│   ├── mtdblock0.bin
│   ├── mtdblock1.bin
│   ├── mtdblock2.bin
│   ├── mtdblock3.bin
│   ├── mtdblock4.bin
│   ├── mtdblock5.bin
│   ├── mtdblock6.bin
│   ├── mtdblock7.bin
│   └── mtd.txt
├── Factory.done
│   ├── config.sh
│   └── factory_test.sh
├── home_y25gam.done
└── newhome
    ├── app
    │   └── script
    │       └── wifidhcp.sh
    └── base
        └── tools
            └── extpkg.sh
```

Should I (re)set my network settings following this guide to be sure: https://github.com/roleoroleo/yi-hack-Allwinner/wiki/Change-WiFi-credentials ?

**Reply by kwint** [reply]
Well, using that guide seemed to work and I can access my camera again via normal means. Weird, but I'll take it!

Unfortunately camera is still on version 0.2.7 instead of 0.3.0, but I'll leave that for now.

**Reply by roleoroleo** [reply]
You could use another sd card to test 0.3.0 and leave 0.2.7 untouched.
